src/components/Autocomplete.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+
+/**
+ * Autocomplete Form Component
+ *
+ * @class Autocomplete
+ * @extends React.Component
+ */
+class Autocomplete extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            lastValue: ''
+        };
+
+        this.handleOnChange = this.handleOnChange.bind(this);
```

Comment:
Create-React-App includes Babel's transform-class-properties plugin installed, so we don't need to do this anymore. Instead we can just use an arrow function for our method definition.

`handleOnChange = e => {}`